main.js

```diff
@@ -1,5 +1,8 @@
 const btn = document.getElementById('btn');
+const pop = document.getElementById('invite');
 const closeButton = document.getElementById('close');
+const closePop = document.getElementById('xx');
```

Comment:
- [ ] Based on requirements all the content of popup modal is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). The JavaScript code should have an array of objects to store JavaScript objects containing the information for each project. Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie: eg:
````javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
````
Please create an object and make the modal popup data populated with the data from the projects array. 😉

Reply:
> * [ ]  Based on requirements all the content of popup modal is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). The JavaScript code should have an array of objects to store JavaScript objects containing the information for each project. Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. Ie: eg:
> 
> 
> ```js
> const projects = [{
>     name: "Submarine game",
>     description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
>     image: "link_to_image.jpg",
>     technologies: ["Phaser3", "JavaScript", "Jest"],
>     link: "link_to_live_version.com",
> }]
> ```
> 
> Please create an object and make the modal popup data populated with the data from the projects array. 😉

- [ ] Please make sure to apply this required change to your project. As the previous reviewer said, all the information in your model should be generated dynamically using JavaScript, so make sure to create an array of objects like the one the previous reviewer mentioned, and then display that information accessing the required object using dot notation (`object.title`) or array-like notation (`object[title]`)

Reply:
- [ ] Please make sure to apply all the requirements asked by the previous reviewer in this [comment.](https://github.com/Yoz21/Portfolio/pull/12/files#r797796306)


https://github.com/Yoz21/Portfolio/blob/80b7d3a285bcd84d0fe6cf8d94bca04e62de1d85/index.html#L103-L235

- [ ] As seen above, all the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.

This means you can create an array of objects to store the project information then populate the `work-card` section of the main page entirely with javascript!

If you are struggling or don't know how to do so, here is an [example](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95)

 - All those cards information should be stored into an array of objects then create a function to render through the array and create dynamically your HTML with the information from the object array, the works section must stay empty